src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -142,15 +142,7 @@ protected IObjectReference(IntPtr thisPtr)
             // pressure rather than tracking all the IObjectReferences that are connected
             // to the same object and only releasing the memory pressure once all of them
             // have been finalized.
-#if NET
-            // Disable on AOT for now until dotnet runtime #104583 is addressed to avoid making the issue happen more often.
-            if (RuntimeFeature.IsDynamicCodeCompiled)
-            {
-                GC.AddMemoryPressure(ComWrappersSupport.GC_PRESSURE_BASE);
-            }
-#else
             GC.AddMemoryPressure(ComWrappersSupport.GC_PRESSURE_BASE);
-#endif
         }
```

Comment:
Also the remove: https://github.com/microsoft/CsWinRT/blob/1844e3ec4ffad4defe6e3d5664d63333e22c49f4/src/WinRT.Runtime/ObjectReference.cs#L398

Reply:
Ah, fixed, thank you!